build.proj

```diff
@@ -18,8 +18,9 @@
 
   <Target Name="BulkRestoreNugetPackages"
           BeforeTargets="Build;BuildAllProjects"
-          DependsOnTargets="_RestoreBuildTools">
+          DependsOnTargets="_RestoreBuildTools"
+          Condition="'$(SolutionFile)'!='' and Exists('$(SolutionFile)')">
```

Comment:
This seems a bit too tolerant.  What valid situation do we have where this won't exist but everything's ok?